### DISCUSSION TITLE
Why is the AI saying non-sequitur? do I need to do something else, or is this the current state AI?

### DISCUSSION THREAD

**Discussion by HubKing** [oobabooga/text-generation-webui#5139] [Q&A] [answered]
I just ran the start script and did not change any settings. In the Model tab, I c&p'ed `facebook/galactica-125m`, clicked the "Download",  and loaded it. I asked simple  things, but the A.I. replied gibberish. I went to HuggingFace and copied the name of the most popular one, `distilgpt2`. I clicked the "Download". I loaded it and also tried to ask simple things. Below is what I got. Utterly useless and dumb. Is this the "AI"? Or it should be this dumb?

> AI
> How can I help you today?
> 
> You
> How do you do?
> 
> AI
> What are your goals for now?
> 
> A: You're going to be able play games in real time as well (you'll need to know that this kind of game will work). So when we go into it, what's next about us doing something interesting or unexpected from our perspective? It seems like people have come up with ways to improve their skills at different levels - but not all these things actually happen on-screen! For example, if someone asks me how much money they earn right away without asking them which way does there really matter whether my income goes towards working so far? We've got some ideas here – maybe one thing would give more motivation than another; perhaps two possibilities could make life easier by giving freebies instead of using just basic tools such Async API. Or even better yet... If somebody says "I am trying", then he might get upset because his goal isn't getting anything done anymore.. But let's say everyone else gets paid less per day… And given those options though, why should anyone want any other means besides having access too many resources over 4 years old?? Well yes no... A lot of developers already start out developing new software platforms before starting off through development :) Why don't expect others to develop apps themselves? Because after 2+years learning languages, once again being inspired makes sense :D To learn language models please read / write code first
> 
> You
> What's your name?
>
> AI
> This guy who was born back home did everything possible while living near friends & family
> 
> You
> What can you do?
> 
> AI
> Yes! Please use Github under Creative Commons license 

![image](https://github.com/oobabooga/text-generation-webui/assets/10900489/4c5bd284-0001-40db-ab05-39dbfc93ae47)

**Comment by TheLounger** [comment, marked as answer]
Both galactica and distilgpt2 are very old and also very small models, it's normal for their output to be rather useless compared to more recent models. Galactica has 125 million params and distilgpt2 has 82 million. But for something more coherent and fun/interesting, you'll want something like 7 billion params, though some very recent 3 billion param models are pretty good as well.

**Reply by HubKing** [reply]
Could you tell me the name of a recommended model in my case? My GPU is Arc A770 that has 16GB VRAM.

**Reply by TheLounger** [reply]
Have a look at some [trending 7B GPTQ models](https://huggingface.co/models?sort=trending&search=7b+gptq), maybe [13B](https://huggingface.co/models?sort=trending&search=13b+gptq) if it's working well. From there on, you could try [larger](https://huggingface.co/models?sort=trending&search=33b+gguf) [GGUF models](https://huggingface.co/models?sort=trending&search=70b+gguf) while offloading some parts of the model (layers) to system RAM.
